src/Chapter3.hs

```diff
@@ -560,22 +632,31 @@ introducing extra newtypes.
 🕯 HINT: if you complete this task properly, you don't need to change the
     implementation of the "hitPlayer" function at all!
 -}
+
+newtype Health = Health Int deriving (Show, Eq)
+newtype Armor = Armor Int deriving (Show, Eq)
+newtype Attack = Attack Int  deriving (Show, Eq)
+newtype Dexterity = Dexterity Int deriving (Show, Eq)
+newtype Strength = Strength Int  deriving (Show, Eq)
+newtype Damage = Damage Int  deriving (Show, Eq)
+newtype Defence = Defence Int deriving (Show, Eq)
+
 data Player = Player
-    { playerHealth    :: Int
-    , playerArmor     :: Int
-    , playerAttack    :: Int
-    , playerDexterity :: Int
-    , playerStrength  :: Int
+    { playerHealth    :: Health
+    , playerArmor     :: Armor
+    , playerAttack    :: Attack
+    , playerDexterity :: Dexterity
+    , playerStrength  :: Strength
     }
 
-calculatePlayerDamage :: Int -> Int -> Int
-calculatePlayerDamage attack strength = attack + strength
+calculatePlayerDamage :: Attack -> Strength -> Damage
+calculatePlayerDamage (Attack attack) (Strength strength) = Damage (attack + strength)
```

Comment:
Nice work on newtypes! :medal_sports: Newtypes are one very important feature in Haskell :+1: